resources/logging/charts/loki/values.yaml

```diff
@@ -285,6 +285,7 @@ kyma:
     logLevel: warn
     lineFormat: json
     removeKeys: "kubernetes, stream"
+    fsBufferSize: 500M
```

Comment:
Isn't the buffer limit 1 GB?

Reply:
Why the loki chart should care at all? That storage attribute should not exist at all in the LogPipeline CRD, does it?

Reply:
Yeah, the fsBufferSize is set globally for all pipelines via an operator flag